test/parallel/test-fs-watch-file-nonexistent.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+var fs = require('fs');
+
+var filename = '/path/to/file/that/does/not/exist';
+
+fs.watchFile(filename, function() {
+  throw new Error('callback should not be called for non-existent files');
```

Comment:
If the functionality is different from 0.10, I think we should at least have something in the docs regarding the difference.